pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/SourceEditorController.java

```diff
@@ -62,6 +64,9 @@
     private ASTTreeItem selectedTreeItem;
     private static final Duration AST_REFRESH_DELAY = Duration.ofMillis(100);
 
+    public static ObservableList<File> auxclasspathFiles;
```

Comment:
I think it would be easier to have here a `Var<List<File>>`. Btw you need to initialise it, eg `Var.newSimpleVar(Collections.emptyList())`.